src/components/App.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 
-
 const App = () => (
   <div id="app">
     <Display result="0" />
```

Comment:
`calculate` should be imported inside this component

Reply:
Hi, I noted this on the pull request description, this is the issue:

 _"Also doesn't contain calculate import on the App component as this throws a linter error because the component is not using the import yet."_

That's why I deleted the import :smile: